Tests/KlaviyoSwiftTests/StateChangePublisherTests.swift

```diff
@@ -80,20 +92,24 @@ final class StateChangePublisherTests: XCTestCase {
                 state.initalizationState = .initialized
                 return StateChangePublisher.test.publisher().eraseToEffect()
             case .flushQueue:
-                state = state
+                return .none
             default:
-                state = state
+                return .none
             }
-            return .none
         }
         
         let reducer = KlaviyoTestReducer(reducer: initializationReducer)
         let test = Store(initialState: .test, reducer: reducer)
         environment.analytics.store = test
-        _ = environment.analytics.store.send(.initialize("foo"))
-        _ = environment.analytics.store.send(.flushQueue)
-        _ = environment.analytics.store.send(.flushQueue)
         
+        @MainActor func runDebouncedEffect() {
```

Comment:
For older versions of swift this is needed to get tests to pass. Noticed this in composable arch code.